src/features/artifacts/components/ArtifactList.tsx

```diff
@@ -12,14 +12,32 @@ import { StyledBox } from "src/styles";
 
 export interface IArtifactsProps {
   /**
-   * @param artifacts list of artifacts
+   * @param build_id build Id
```

Comment:
We should aim for somewhat more descriptive doc strings. The above isn't super helpful

Reply:
Can I remove the description? The name is quite descriptive.

Reply:
something like "The id of the build we want to display the artifacts for"